source/features/minimize-upload-bar.css

```diff
@@ -5,5 +5,4 @@
 
 .rgh-minimize-upload-bar .js-upload-markdown-image.is-default textarea {
 	border-bottom-style: solid;
-	border-radius: 3px;
```

Comment:
Also needed. Look at the bottom corners

<img src="https://user-images.githubusercontent.com/1402241/86591102-2fb86c00-bf91-11ea-9f4c-6c88ebd4edf7.png">

Reply:
That's weird, I remember not seeing this:

```css
.upload-enabled textarea {
  border-bottom-right-radius: 0;
  border-bottom-left-radius: 0;
}
```